internal/ogtags/cache.go

```diff
@@ -13,6 +13,10 @@ func (c *OGTagCache) GetOGTags(url *url.URL, originalHost string) (map[string]st
 		return nil, errors.New("nil URL provided, cannot fetch OG tags")
 	}
 
+	if len(c.ogDefault) != 0 {
```

Comment:
The downside is this limits per-route default syntax but that's out of scope here